app/models/atmosphere/virtual_machine_template.rb

```diff
@@ -157,7 +157,7 @@ def perform_delete_in_cloud
     # Returns the hourly cost for this template assuming a given VM flavor
     def get_hourly_cost_for(flavor)
       incarnation = flavor.flavor_os_families.
-                    find_by(os_family: appliance_type.os_family)
+                    detect{|fof| fof.os_family == appliance_type.os_family}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.